modules/database/funcs.go

```diff
@@ -102,6 +103,7 @@ func (d *Database) GetDatabase() *mariadbv1.MariaDBDatabase {
 //
 func (d *Database) CreateOrPatchDB(
 	ctx context.Context,
+	instance *keystonev1.KeystoneAPI,
```

Comment:
we can not do that. we can not pass in a crd specific instance. the database functions are used by all the operators. the helper has the object if it is needed.

Reply:
ok